### PR TITLE
Add Movinet to action classification

### PR DESCRIPTION
**Pull request by jaegukhyun**
### Summary
Add MoViNet model from OTX1.x

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: tests/unit/algo/action_classification/heads/test_movinet_head.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from otx.algo.action_classification.heads.movinet_head import MoViNetHead
+
+
+class TestMoViNetHead:
+    @pytest.fixture()
+    def fxt_movinet_head(self) -> MoViNetHead:
+        return MoViNetHead(
+            5,
+            480,
+            2048,
```
Comment (vinnamkim) [inline tests/unit/algo/action_classification/heads/test_movinet_head.py]: Is there any reason to use such a high dimension in unit tests?

File: tests/unit/algo/action_classification/backbones/test_movinet.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+class TestMoViNet:
+    @pytest.fixture()
+    def fxt_movinet(self) -> OTXMoViNet:
+        return OTXMoViNet()
+
+    def test_forward(self, fxt_movinet: OTXMoViNet) -> None:
+        x = torch.randn(1, 3, 8, 224, 224)
+        assert fxt_movinet(x).shape == torch.Size([1, 480, 1, 1, 1])
```
Comment (vinnamkim) [inline tests/unit/algo/action_classification/backbones/test_movinet.py:17]: It seems not the tensor shape which comes from the head output. What's the reason?

File: tests/unit/algo/action_classification/heads/test_movinet_head.py (hunk 1 above)
Comment (jaegukhyun) [reply tests/unit/algo/action_classification/heads/test_movinet_head.py]: Reduced by 
e904e21

File: tests/unit/algo/action_classification/backbones/test_movinet.py (hunk 2 above)
Comment (jaegukhyun) [reply tests/unit/algo/action_classification/backbones/test_movinet.py:17]: This test is for testing backbone class. This MoViNet backbone class and MoViNet head class constitute overall 3d recognizer.

**Review by vinnamkim — APPROVED** [review]

**Review by harimkang — APPROVED** [review]

**Review by eunwoosh — APPROVED** [review]

**Pull request merged**